src/lambda-edge/shared/validate-jwt.ts

```diff
@@ -42,9 +42,9 @@ export async function validate(jwtToken: string, jwksUri: string, issuer: string
         issuer,
         ignoreExpiration: false,
     };
-    return new Promise((resolve, reject) => verify(
+    return new Promise<any>((resolve, reject) => verify(
```

Comment:
Instead of any I think there is a nice type/interface for decoded tokens in the jsonwebtoken typings?

Reply:
I looked and it doesn't seem like it:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/28ab593ab33d65a7a70ea7bc51163fe9779930cd/types/jsonwebtoken/index.d.ts#L103

I went with a type of `{ [key: string]: any } | undefined` which is close to what the jsonwebtoken typings is sending in the verify callback. More details are in a comment in the forthcoming code.